packages/gitmoji-changelog-cli/src/cli.js

```diff
@@ -31,6 +49,9 @@ async function main(options = {}) {
   } catch (e) {
     logger.error(e)
   }
+
+  // force quit (if the latest version request is pending, we don't wait for it)
```

Comment:
Yeah I should test that process.exit is triggered.
But I would let the comment too.

Reply:
Why since the information is present in the test ?

Reply:
Because if I read `process.exit(0)` in some code, I would find this weird.

And I think I won't look into to test to understand why this is here.

I can rename the comment:
`// if the latest version request is pending, we don't wait for it`

But I think this is interesting to know WHY we put a process.exit there